roles/dnsmasq/handlers/main.yml

```diff
@@ -7,4 +17,18 @@
       networking
       {%- endif %}
     state: restarted
-  when: ansible_os_family != "CoreOS"
+  register: network_runner
+  async: 600
+  poll: 0
+
+- name: Dnsmasq | Wait for network reloaded
+  vars:
+    ansible_ssh_pipelining: yes
+  async_status: jid={{ network_runner.ansible_job_id }}
+  register: network_jobs
+  until: network_jobs.finished
+  retries: 30
+
+- name: Dnsmasq | update resolvconf maybe
+  command: /sbin/resolvconf -u
+  ignore_errors: true
```

Comment:
For rhel family, there is no this handler, yet (because it cannot pass CI).
So the ```resolvconf -u``` code in task is preserved.
But I can't put that ```which resolvconf`` selector logic from the task as facts to the handler!
So if it is not there, ingore_errors will allow that one to pass. Herefrom, a "maybe" part.

note: the handler is ok but for now it is just not called for rhel case
Once fixed the strange issues with failing rhel/centos after network restart, we could fix the task to allow this handler for rhel as well, leaving the handler code untouched!